doc/reference/tooling/tt_cli/cluster.rst

```diff
@@ -1,10 +1,242 @@
+To write a configuration to a local ``config.yaml``, run ``tt cluster publish``
+with two arguments:
+
+*   the application name.
+*   the path to a YAML file from which the configuration should be taken.
+
+.. code-block:: console
+
+    $ tt cluster publish myapp source.yaml
+
+To print a local configuration from ``config.yaml`` in an application directory,
+run ``tt cluster show`` with the application name:
+
+.. code-block:: console
+
+    $ tt cluster show myapp
+
+.. _tt-cluster-centralized:
+
+Managing configurations in centralized storages
+-----------------------------------------------
+
+``tt cluster`` can manage centralized cluster configurations in storages of both
+supported types: :ref:`etcd <configuration_etcd>` or a Tarantool-based configuration storage.
+
+To send a configuration from a file to a centralized configuration storage,
+run ``tt cluster publish`` with a URI of this storage's
+instance as the target. For example, to send a configuration from ``source.yaml``
+to a local etcd instance on the default port ``2379``:
+
+.. code-block:: console
+
+    $ tt cluster publish "http://localhost:2379/myapp" source.yaml
+
+A URI must include a prefix that is unique for the application. It can also include
+credentials and other connection parameters. Find the detailed description of the
+URI format in :ref:`tt-cluster-centralized-uri`.
+
+To print a cluster configuration from a centralized storage, run ``tt cluster show``
+with a storage URI including the prefix identifying the application. For example, to print
+``myapp``'s configuration from a local etcd storage:
+
+.. code-block:: console
+
+    $ tt cluster show "http://localhost:2379/myapp"
+
+.. _tt-cluster-centralized-authentication:
+
+Authentication
+~~~~~~~~~~~~~~
+
+There are three ways to pass the credentials for connecting to the centralized configuration storage:
+
+*   Environment variables ``TT_CLI_ETCD_USERNAME`` and ``TT_CLI_ETCD_PASSWORD``:
+
+    .. code-block:: console
+
+            $ export TT_CLI_ETCD_USERNAME=myuser
+            $ export TT_CLI_ETCD_PASSWORD=p4$$w0rD
+            $ tt cluster show "http://localhost:2379/myapp"
+
+*   ``tt cluster`` options ``-u``/``--username`` and ``-p``/``--password``:
+
+    .. code-block:: console
+
+        $ tt cluster show "http://localhost:2379/myapp" -u myuser -p p4$$w0rD
+
+*   Credentials specified in the storage URI: ``https://username:password@host:port/prefix``:
+
+    .. code-block:: console
+
+        $ tt cluster show "http://myuser:p4$$w0rD@localhost:2379/myapp"
+
+They are applied with the following precedence, from highest to lowest:
+
+*   URI credentials.
+*   ``tt cluster`` options.
+*   Environment variables.
+
+If connection encryption is enabled on the configuration storage, pass the required
+SSL parameters in the :ref:`URI arguments <tt-cluster-centralized-uri>`.
+
+.. _tt-cluster-centralized-uri:
+
+URI format
+~~~~~~~~~~
+
+A URI of the cluster configuration storage has the following format:
```

Comment:
An idea from the top of my head (maybe I'm wrong here): move the entire `URI format` section to reference and keep only How-to info here. So, the reference might include two levels:

- Reference
   - Options
   - URI format

Reply:
The idea is generally OK but I don't see much benefit.